content/post/sp-reflection-1.md

```diff
@@ -0,0 +1,21 @@
++++
+title = 'Reflection #1'
+date = 2024-05-23T20:22:10-04:00
+authors = ["Serena Peruzzo"]
++++
+
+We started a summer fellowship!  <!--more-->
+
+The 1RG summer fellowship is the brain child of Xavier, Max and I. We are manifesting it by establishing peer-to-peer mentorship rituals, and accountability systems in order to spend the next few months researching topics that interests us.
+
+There are a few reasons that pull me towards this experiment:
+1. I wanted to spend more time thinking and writing, but often find it hard to prioritize it and let it slide to the bottom of my TODOs
+2. I have experienced academia as a soul crushing machine and found it impossible to pursue research in that context, despite being very drawn to the exploratory aspects of it
+3. My experience at the recurse centre, a self directed retreat for curious programmers, was life changing and what originally drove me to go back to academia.
```

Comment:
For *my* research I definitely want to have a conversation with you about what was life-changing about Recurse and what aspects of it you think I might learn from.